recipe/meta.yaml

```diff
@@ -1,17 +1,13 @@
-{%set name = "nltk_data" %}
-{%set version = "2017.10.22" %}
+{% set version = "2017.10.22" %}
```

Comment:
What is the datever for this commit? Can you update it here?

Reply:
I grabbed the latest commit and updated the detever for that commit. I also updated the hash to fit the hash of that tar file.

Reply:
Only thing I am not sure about is if I need to require the newest version of the `nltk` package to go along with the latest version of the data.

Reply:
> I grabbed the latest commit and updated the detever for that commit. I also updated the hash to fit the hash of that tar file.

Thanks!

> Only thing I am not sure about is if I need to require the newest version of the `nltk` package to go along with the latest version of the data.

Probably not but having that won't hurt users.